apps/nextjs-website/src/app/page.tsx

```diff
@@ -30,13 +29,11 @@ const NotSsrWebinarsSection = dynamic(
 );
 
 const Home = async () => {
-  const webinars = await getVisibleInHomeWebinars();
-
   const homepage = await getHomepageProps();
 
   return (
     <>
-      <NotSsrWebinarHeaderBanner webinars={webinars} />
+      <NotSsrWebinarHeaderBanner webinars={[...homepage.webinars]} />
```

Comment:
I cannot see the banner even if i properly set in Strapi a webinar in the future